node/src/node_configurator/node_configurator_standard.rs

```diff
@@ -635,26 +635,14 @@ pub mod standard {
 
     pub fn get_db_password(
         multi_config: &MultiConfig,
-        streams: &mut StdStreams,
+        _streams: &mut StdStreams,
         config: &mut BootstrapperConfig,
         persistent_config: &mut dyn PersistentConfiguration,
     ) -> Result<Option<String>, ConfiguratorError> {
         if let Some(db_password) = &config.db_password_opt {
             return Ok(Some(db_password.clone()));
         }
-        let db_password_opt = match value_user_specified_m!(multi_config, "db-password", String) {
```

Comment:
Also, I'd check out whether value_user_specified_m! is still being used somewhere else after you removed it here. It's quite an unusual macro so it's possible that we don't need it existing at all.